modules/troubleshooting-disabling-autoreboot-mco.adoc

```diff
@@ -5,7 +5,16 @@
 [id="troubleshooting-disabling-autoreboot-mco_{context}"]
 = Disabling Machine Config Operator from automatically rebooting
 
-When configuration changes are made by the Machine Config Operator, {op-system-first} must reboot for the changes to take effect. Whether the configuration change is automatic, such as when a `kube-apiserver-to-kubelet-signer` CA is rotated, or manual, such as when a registry or SSH key is updated, an {op-system} node reboots automatically unless it is paused.
+When configuration changes are made by the Machine Config Operator, {op-system-first} must reboot for the changes to take effect. Whether the configuration change is automatic, such as when a `kube-apiserver-to-kubelet-signer` CA is rotated, or manual, an {op-system} node reboots automatically unless it is paused.
+
+[NOTE]
+====
+The following modifications do not trigger a node reboot:
+
+* SSH key updates: modifications to the `spec.config.ignition.passwd.users.sshAuthorizedKeys` parameter in a machine config
+* pull secret updates: modifications to the global pull secret or pull secret in the `openshift-config` namespace
+* `registries.conf` file updates: modifications that affect the `/etc/containers/registries.conf` file
```

Comment:
... e.g. ICSP changes

Reply:
@yuqi-zhang Thanks for the comments.  
What is ICSP? We don't mention it in the docs apparently.

Reply:
Ah sorry that would be imageContentSourcePolicy, e.g. here: https://docs.openshift.com/container-platform/4.6/updating/updating-restricted-network-cluster.html#images-configuration-registry-mirror_updating-restricted-network-cluster

i.e. if you create/update an imagecontentsourcepolicy object, it gets populated into a machineconfig. This was one of the asks from customers so I thought it might have made sense to explicitly list it here. I defer to your judgement on that though.